### PR TITLE
Fixing log spam caused by stray Synchronized annotations (and adding …

### PR DESCRIPTION
**Pull request by sanaerosen-yelp**
…a readme change that got missed in the last CR)

### CODE REVIEW

File: buildSrc/src/main/java/com/yelp/gradle/varanus/GlobalDependencies.kt [hunk 1]
```diff
@@ -80,6 +80,6 @@ object BuildScriptLibs {
 
 object Publishing {
     const val GROUP = "com.yelp.android"
-    const val VERSION = "2.0.2"
+    const val VERSION = "2.0.4"
```
Comment (Kurt-Bonatz) [inline buildSrc/src/main/java/com/yelp/gradle/varanus/GlobalDependencies.kt]: Is there a reason we're skipping 3?

**Review by Kurt-Bonatz — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/java/com/yelp/gradle/varanus/GlobalDependencies.kt (hunk 1 above)
Comment (sanaerosen-yelp) [reply buildSrc/src/main/java/com/yelp/gradle/varanus/GlobalDependencies.kt]: For some reason it's at 2.0.3 internally, I think I bumped the version to try and fix something

Comment (sanaerosen-yelp) [reply buildSrc/src/main/java/com/yelp/gradle/varanus/GlobalDependencies.kt]: with regards to publishing internally, not in the library itself

**Review by martinpdd8 — APPROVED** [review]
The AtomicInteger changes look good to me!

**Pull request merged**